chat.go

```diff
@@ -21,9 +22,15 @@ var (
 	ErrChatCompletionStreamNotSupported = errors.New("streaming is not supported with this method, please use CreateChatCompletionStream")              //nolint:lll
 )
 
+type EscapeString string
+
+func (esc EscapeString) MarshalJSON() ([]byte, error) {
+	return []byte(strconv.QuoteToASCII(string(esc))), nil
+}
+
 type ChatCompletionMessage struct {
-	Role    string `json:"role"`
-	Content string `json:"content"`
+	Role    string       `json:"role"`
+	Content EscapeString `json:"content"`
```

Comment:
Can you adopt a method that doesn't involve breaking changes?

Reply:
I am not very familiar with other implementation methods.